images/macos/provision/core/openssl.sh

```diff
@@ -1,33 +1,6 @@
 #!/bin/sh
 
-source ~/utils/utils.sh
-
-echo "Installing OpenSSL..."
-export PATH="/usr/local/bin:/usr/local/sbin:~/bin:$PATH"
-
 echo Installing OpenSSL...
 /usr/local/bin/brew install openssl
```

Comment:
Why is brew used with path here, but it's used without path below (`brew --prefix openssl`)?

Reply:
I don't know :)
Changed to ` ln -sf $(/usr/local/bin/brew --prefix openssl)/bin/openssl /usr/local/bin/openssl`